jetty-server/src/main/java/org/eclipse/jetty/server/HttpInput.java

```diff
@@ -150,10 +151,9 @@ public boolean consumeAll()
     {
         try (AutoLock lock = _contentProducer.lock())
         {
-            IOException failure = new IOException("Unconsumed content");
             if (LOG.isDebugEnabled())
-                LOG.debug("consumeAll {}", this, failure);
-            boolean atEof = _contentProducer.consumeAll(failure);
+                LOG.debug("consumeAll {}", this);
+            boolean atEof = _contentProducer.consumeAll(CONSUME_ALL_IOEXCEPTION);
```

Comment:
Sure.

Reply:
Looking more carefully at this code, since `consumeAll()` is ran under a lock we could simply have a field that we initialize in `consumeAll()` with a simple `if (null) new Exception` pattern. WDYT?